lang/en/docs/tutorials/dft/optical/epsilon-optimal-basis.md

```diff
@@ -0,0 +1,124 @@
+# Optical property with optimal basis
+
+In this tutorial we will go through the details of optical property calculation
+with optimal basis function. We will use Quantum ESPRESSO SIMPLE.X program to
+calculate dielectric function of silicon. Please refer to the original paper:
+[SIMPLE code: Optical properties with optimal basis functions, Prandini, G.,
+Galante, M., Marzari, N., & Umari, P., Computer Physics Communications, **240**,
+106 (2019)](https://doi.org/10.1016/j.cpc.2019.02.016) for the detailed physics
+behind this calculation.
+
+Below we will replicate the [example 5 from Quantum ESPRESSO GWW directory](
+https://gitlab.com/QEF/q-e/-/tree/qe-7.3/GWW/examples/). Alternatively, input
+and reference output files are available in our [CLI-job-examples](
+https://github.com/Exabyte-io/cli-job-examples/tree/main/espresso/simple.x)
+repository.
+
+
+## Workflow
+
+### a. PW SCF calculation
```

Comment:
Let's use numbers like in https://docs.mat3ra.com/tutorials/ml/deepmd-mlff-with-espresso-cp-and-lammps/

And let's update the Internal.md to put guidelines on how this should we

Reply:
If we want to enumerate sections/sub-sections in all pages, we could do it automatically with tools like https://github.com/timvink/mkdocs-enumerate-headings-plugin